src/mono/netcore/System.Private.CoreLib/src/System/Threading/TimerQueue.Browser.cs

```diff
@@ -0,0 +1,127 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Runtime.CompilerServices;
+
+namespace System.Threading
+{
+    // Runtime interface
+    internal static class WasmRuntime
+    {
+        private static Dictionary<int, Action> callbacks = new Dictionary<int, Action>();
+        private static int next_id;
```

Comment:
Nit: Elsewhere in the codebase we've tried to use the s_ prefix for statics, at least for newly written code.